submissions/Lu-sure/post-oop/index.js

```diff
@@ -0,0 +1,140 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+    constructor(name, age) {
+      this.name = name;
+      this.age = age;
+      this.species;
+    }
+
+    introduce() {
+      return `I\'ve existed for ${this.age} years`
+    }
+  }
+
+  class Human extends Inhabitant {
+    constructor(options) {
+      super(options.name, options.age);
+      this.name = `${options.name} ${options.lastName}`;
+      this.species = 'human';
+      this.saying = options.saying;
+    }
+
+    introduce() {
+      return `${super.introduce()} as a ${this.species}. What I can say, \"${this.saying}\"`;
+    }
+  }
+
+  class Woman extends Human {
+    constructor(options) {
+      super(options);
+      this.gender = 'female';
+      this.lovedBy;
+    }
+
+    beLoved() {
+      return `\"I\'m loved by ${this.lovedBy == this.name ? 'myself' : this.lovedBy}.\"`;
```

Comment:
Template string literals do not requre `"` or `'` escaped.

Reply:
Fixed.